deeplearning4j-nn/src/main/java/org/deeplearning4j/eval/EvaluationUtils.java

```diff
@@ -14,6 +14,133 @@
  */
 public class EvaluationUtils {
```

Comment:
this is something I actually didn't write, but was somehow picked up in mvn formatter :(